src/server/check-answers/controller.js

```diff
@@ -21,9 +22,13 @@ export const checkAnswersGetController = {
       license: License.fromState(req.yar.get('license'))
     }
 
-    const isValid = Object.values(tasks).every(
-      (task) => task.validate().isValid
-    )
+    // TODO see how to get the whole state in one go with req.yar.get('application')
```

Comment:
Looked at the state in redis and it is under an "item" property but couldn't get that one so maybe we have to nest everything under another property?

Reply:
@joseluisgraa  - I didn't want to pile in too much change at once, but I definitely thought that on `application` state that had the sections beneathe it ultimately makes sense. 

One for after this round of refactors & a bit of feature implementation?

Reply:
yup, exactly that :)